docs/en_US/ExperimentConfig.md

```diff
@@ -392,6 +392,12 @@ machineList:
 
       __image__ set the image to be used in __worker__.
 
+* __localConfig__
+  __localConfig__ is applicable only if __trainingServicePlatform__ is set to ```local```, otherwise there should not be __localConfig__ section in configuration file.
```

Comment:
There should be an empty between line 395 and line 396, otherwise the markdown file will not change lines.

Reply:
Thanks, updated.